Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -620,7 +620,7 @@
     <string name="settings_notifications_illustration_description_inactive">"Eine Frau hat die Mitteilungen ihrer Corona-Warn-App ausgeschaltet."</string>
     <!-- XBUT: settings - go to reset application -->
     <string name="settings_reset_title">"Anwendung zurücksetzen"</string>
-    <!-- XTXT: settings(reset) - explains the user what do expect when he navigates to reset -->
+    <!-- XTXT: settings(reset) - explains the user what do expect when she or he navigates to reset -->
```

Comment:
To reduce merge conflicts only update something in german, it will be shipped to the other languages with the next translation system. 
Maybe we could get ride completely of genders in this spot:
```suggestion
    <!-- XTXT: settings(reset) - explains the user what to expect when navigating to the reset settings -->
```